contracts/oss.yml

```diff
@@ -14,9 +14,20 @@ tags:
       permissions for organization resources and provides an API token for authentication.
       An authorization belongs to an organization and only contains permissions for that organization.
 
-      An authorization is only visible to the user that created it.
-      Optionally, when creating an authorization, you can scope it to a specific user.
-      A _user session_ carries all the permissions granted by all the user's authorizations.
+      #### Limitations
+
+      API tokens are visible to the user who created the authorization and to any
+      user with an _[operator token](https://docs.influxdata.com/influxdb/v2.3/security/tokens/#operator-token))_.
+      In InfluxDB OSS, even if an API token has `read-authorizations` permission, the
+      token can't be used to view its authorization details.
```

Comment:
this reads a bit confusing--what does `read-authorizations` let ppl do if not view authorization details? Does this mean you can see tokens, but not permissions for the token?